database/index.js

```diff
@@ -0,0 +1,81 @@
+const mongoose = require('mongoose');
+const db = mongoose.connection;
+
+mongoose.connect('mongodb://localhost/recommended', {useNewUrlParser: true});
+
+db.on('error', function() {
+  console.log('Could not connect to MongoDB');
+});
+
+db.once('open', function() {
+  console.log('Connected to MongoDB');
+});
+
+const imageSchema = new mongoose.Schema({
+  key: Number,
+  recommended: Array,
+  raw: String,
+  full: String,
+  regular: String,
+  small: String,
+  thumb: String
+});
+
+const Image = mongoose.model('image', imageSchema);
+
+const saveImages = (images => {
+  const imagesSaved = [];
+
+  for (let i = 0; i < images.length; i++) {
+    const newImage = new Image({
+      key: i + 1,
+      recommended: randomProjects(),
+      raw: images[i].raw,
+      full: images[i].full,
+      regular: images[i].regular,
```

Comment:
If you're no longer using these properties, I would go ahead and get rid of them.